core/src/main/scala/org/apache/spark/storage/BlockManager.scala

```diff
@@ -684,6 +684,7 @@ private[spark] class BlockManager(
     if (blockId.isShuffle) {
       logDebug(s"Putting shuffle block ${blockId}")
       try {
+
```

Comment:
Shall we avoid touching irrelevant file?

Reply:
Sure